packages/ToggleSwitch/ToggleSwitch.jsx

```diff
@@ -150,16 +111,16 @@ ToggleSwitch.propTypes = {
   /** Language provided to the copy prop in TDS ToolTip (en, fr). */
   toolTipCopy: PropTypes.string,
```

Comment:
Can we make `toolTipCopy` a required prop?

Reply:
Done